components/system/CodeBlock.js

```diff
@@ -27,6 +24,98 @@ const STYLES_CODE_BLOCK = css`
       height: 0;
     }
   }
+
+  :not(pre) > code,
```

Comment:
So I think a key thing with this next iteration of the CodeBlock is to inline all of the styles for prism instead of using  global styles or scoped styles. So I'm going to have to block this PR on that (Sorry it took me so long to get to it).

Reply:
Sure. Not a problem. I guess I did not understand clearly was the demands were.